e-notary-server/routes/usign/authToken.js

```diff
@@ -0,0 +1,22 @@
+var axios = require('axios');
+var express = require('express');
+var router = express.Router();
+
+router.post("/", (req, res, next) => {
+    var config = {
+        headers: {
+            "Authorization": `Bearer ${req.body.token}`
+        }
+    }
+    var data = "";
+    var url = "https://api.usign.kr:18443/PDFsign/auth/verifyToken/";
+    axios.post(url, data, config)
+        .then((apiRes) => {
+            res.send(apiRes.data)
+        })
+        .catch((err) => {
+            console.log(err)
```

Comment:
if there is an error, check if it is internal server error or not, and still send a response with the appropriate information regarding the error